interfaces/Portalicious/src/app/pages/project-registrations/components/export-registrations/export-registrations.component.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  inject,
+  input,
+  model,
+  signal,
+  ViewChild,
+} from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+import {
+  injectMutation,
+  injectQuery,
+} from '@tanstack/angular-query-experimental';
+import { MenuItem } from 'primeng/api';
+import { CalendarModule } from 'primeng/calendar';
+import { FloatLabelModule } from 'primeng/floatlabel';
+
+import { FinancialServiceProviderName } from '@121-service/src/financial-service-providers/enum/financial-service-provider-name.enum';
+import { ExportType } from '@121-service/src/metrics/enum/export-type.enum';
+import { PermissionEnum } from '@121-service/src/user/enum/permission.enum';
+
+import { ButtonMenuComponent } from '~/components/button-menu/button-menu.component';
+import { ConfirmationDialogComponent } from '~/components/confirmation-dialog/confirmation-dialog.component';
+import { FormFieldWrapperComponent } from '~/components/form-field-wrapper/form-field-wrapper.component';
+import { ProjectApiService } from '~/domains/project/project.api.service';
+import { Registration } from '~/domains/registration/registration.model';
+import { LatestExportDateComponent } from '~/pages/project-registrations/components/latest-export-date/latest-export-date.component';
+import { AuthService } from '~/services/auth.service';
+import { ExportService } from '~/services/export.service';
+import { ActionDataWithPaginateQuery } from '~/services/paginate-query.service';
+import { ToastService } from '~/services/toast.service';
+
+@Component({
+  selector: 'app-export-registrations',
+  standalone: true,
+  imports: [
+    ConfirmationDialogComponent,
+    LatestExportDateComponent,
+    ButtonMenuComponent,
+    CalendarModule,
+    FloatLabelModule,
+    FormsModule,
+    FormFieldWrapperComponent,
+  ],
+  templateUrl: './export-registrations.component.html',
+  styles: ``,
+  providers: [ToastService],
```

Comment:
Doesn't need to be here as we're using inject()

Reply:
I don't understand why....but....it does 😅 

I think it has something to do with how the underlying `MessageService` works. But if we don't add the `ToastService` to the `providers`, angular complains.

Maybe worth (separately) doing some digging to understand why this is happening. But for now, I've just done this everywhere it was needed.